flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/TableStoreFactoryOptions.java

```diff
@@ -34,6 +35,6 @@
     public static final ConfigOption<String> LOG_SYSTEM =
             ConfigOptions.key("log.system")
                     .stringType()
-                    .defaultValue("kafka")
+                    .noDefaultValue()
```

Comment:
why remove `defaultValue` ?

Reply:
`noDefaultValue` means no log system by default. File store continuous reading is used by default.